src/registries/autofill_modifiers.ts

```diff
@@ -19,6 +20,25 @@ import { Registry } from "./registry";
 export const autofillModifiersRegistry = new Registry<AutofillModifierImplementation>();
 
 autofillModifiersRegistry
+  .add("ALPHANUMERIC_INCREMENT_MODIFIER", {
+    apply: (rule: AlphanumericIncrementModifier, data: AutofillData) => {
+      rule.current += rule.increment;
+      const prefix = rule.prefix;
+      const content = `${prefix}${rule.current
+        .toString()
+        .padStart(rule.numberPostfixLength || 0, "0")}`;
+      const tooltipValue = rule.prefix ? content : formatValue(rule.current, data.cell?.format);
```

Comment:
does formatting with the format make sense ? Shouldn't it be always `content`?
prefix should always be truthy, right ? The autofill rule itself doesn't make sense